src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelServiceTest.kt

```diff
@@ -93,4 +105,49 @@ class EndretUtbetalingAndelServiceTest {
         verify(exactly = 1) { vilkårsvurderingService.hentAktivVilkårsvurderingForBehandling(behandling.id) }
         verify(exactly = 1) { beregningService.oppdaterTilkjentYtelsePåBehandling(behandling, any(), any()) }
     }
+
+    @Test
+    fun `hentSanityBegrunnelserMedEndringsårsak - skal returnere et map med begrunnelsestyper mappet mot liste av begrunnelser`() {
+        // Arrange
+        every { sanityService.hentSanityBegrunnelser() } returns
+            listOf(
+                SanityBegrunnelse(
+                    NasjonalEllerFellesBegrunnelse.AVSLAG_ENDRINGSPERIODE_ALLEREDE_UTBETALT_ANNEN_FORELDER.sanityApiNavn,
+                    "avslagAlleredeUtbataltAnnenForelderEndringsperiode",
+                    SanityBegrunnelseType.ENDRINGSPERIODE,
+                    Vilkår.entries.toList(),
+                    rolle = emptyList(),
+                    triggere = emptyList(),
+                    utdypendeVilkårsvurderinger = emptyList(),
+                    hjemler = emptyList(),
+                    endretUtbetalingsperiode = emptyList(),
+                    endringsårsaker = listOf(Årsak.ALLEREDE_UTBETALT),
+                    resultat = SanityResultat.AVSLAG,
+                    skalAlltidVises = false,
+                    støtterFritekst = false,
+                ),
+                SanityBegrunnelse(
+                    NasjonalEllerFellesBegrunnelse.AVSLAG_ENDRINGSPERIODE_ALLEREDE_UTBETALT_SØKER.sanityApiNavn,
+                    "avslagAlleredeUtbetaltSokerEndringsperiode",
+                    SanityBegrunnelseType.ENDRINGSPERIODE,
+                    Vilkår.entries.toList(),
+                    rolle = emptyList(),
+                    triggere = emptyList(),
+                    utdypendeVilkårsvurderinger = emptyList(),
+                    hjemler = emptyList(),
+                    endretUtbetalingsperiode = emptyList(),
+                    endringsårsaker = listOf(Årsak.ALLEREDE_UTBETALT),
+                    resultat = SanityResultat.AVSLAG,
+                    skalAlltidVises = false,
+                    støtterFritekst = false,
+                ),
+            )
+
+        // Act
+        val endringsårsakbegrunnelser = endretUtbetalingAndelService.hentSanityBegrunnelserMedEndringsårsak()
+
+        // Assert
+        assertEquals(9, endringsårsakbegrunnelser.size)
```

Comment:
Kan bruke `.hasSize(...)` metoden her 

```suggestion
        assertThat(endringsårsakbegrunnelser).hasSize(9)
```